.github/changeset-version.js

```diff
@@ -22,7 +26,26 @@ function parseVersion(version) {
 	return [parseInt(match[1]), parseInt(match[2]), parseInt(match[3])];
 }
 
-const miniflarePath = path.resolve(__dirname, "../packages/miniflare");
+/**
+ * Rewrites the specified file line-by-line.
+ * @param {string} filePath
+ * @param {(line: string) => string} transformer
+ */
+async function transformFile(filePath, transformer) {
+	const tmpFile = path.join(os.tmpdir(), `transform-${crypto.randomUUID()}`);
+	const input = fs.createReadStream(filePath);
+	const output = fs.createWriteStream(tmpFile);
+	const rl = readline.createInterface({ input });
+	for await (const line of rl) output.write(`${transformer(line)}\n`);
```

Comment:
Seems like this async iterator has caused a spread of async culminating in a `void main()`

Considering the only use here is for the CHANGELOG.md file, currently ~0.5MB, I would argue we could load that into memory without issues (this isn't a server with a need to process other things concurrently) – the simplicity of reading the whole file in, transforming it, and writing back out trumps the memory optimisation